### PR TITLE
Add new recordings to the readme page

### PR DESCRIPTION
**Pull request by fgiloux**
Signed-off-by: Frederic Giloux <fgiloux@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,31 @@
 # Pipeline Service
 
-Pipeline Service provides a SaaS for pipelines. It leverages Kubernetes, Tekton, Tekton Triggers, Tekton Results, Pipelines as Code together with kcp.
+Pipeline Service provides a SaaS for pipelines. It leverages:
+
+- kcp for serving the API and acting as control plane
+- Kubernetes / OpenShift for the compute
+- Tekton Pipelines, Results and friends for the core of the service
+- OpenShift GitOps / Argo CD, Pipelines as Code for managing the infrastructure
```
Comment (Roming22) [inline README.md:8]: Isn't Pipelines as Code also use for the core service? My understanding was that it was providing the service to enable hooks in git repos (with the help of hcg).

File: README.md [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
+
 
 ## Why Pipeline Service?
 
 Tekton and Kubernetes provide a great infrastructure for build pipelines. They come however with some limitations.
 
-- Multi-tenancy: Kubernetes provides a level of multi-tenancy. CustomResourceDefinitions are however cluster scoped. This induces a coupling between the operator version provided by the platform and what you can use as a tenant. The control plane is also shared between tenants.
+- Multi-tenancy: Kubernetes provides a level of multi-tenancy. This does however not extend to cluster scoped resources. CustomResourceDefinitions (CRD) are extensively used for extending the Kubernetes API, following [the operator pattnern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).  CRDs are cluster scoped. This induces a coupling between the operator version provided by the platform and what you can use as a tenant. The control plane is also shared between tenants.
```
Comment (Roming22) [inline README.md:15]: `This does however not extend to` -> `However this does not extend to`
`pattnern` -> `pattern`

File: README.md [hunk 3]
```diff
@@ -1,19 +1,31 @@
...
 - Scalability: Kubernetes has made it easy to distribute the load onto many servers and scalability at cloud scale more approachable. Like everything, its control plane has however its limits.
 - Availability and geo-redundancy: Kubernetes control plane is based on an etcd cluster, which is sensible to latency between its members. This restricts what can be done in terms of geographical distribution.
 
 kcp helps with mitigating these challenges, pushing the limits to new horizons.
 
 Pipelines as Code is the veneer that brings to the users a great experience directly from their git repository.
 
+**kcp - workspace isolation** (~3 mns)
```
Comment (Roming22) [inline README.md:23]: Abbreviation for minutes is `min` and is invariable.

File: README.md (hunk 1 above)
Comment (fgiloux) [reply README.md:8]: You are correct. We use it and provide it. There is a bit of the chicken and egg story.

 I was purposely vague on the core of the service as the list is not exhaustive and may further extend, As you know we are adding Tekton chain for instance. One can understand Pipelines as Code as part of Tekton and friends.
 On the other side I wanted to emphasize that we are not only putting components together but also providing the way to manage a cohesive platform. Therefore we leverage GitOps and Pipelines as Code.
 
 I am open to proposals for a better way of formulating it. It has however to be concise. In ~1mn the reader should get a clue of the value proposal of the project and the approach taken.

**Review by ramessesii2 — APPROVED** [review]
Looks good to me.

**Comment by bnallapeta** [comment]
@Roming22 Looks like the Openshift-ci bot has merged the PR with @ramessesii2's approval. If you have comments that require changes, please revert the PR so that @fgiloux can work on them.

**Comment by xinredhat** [comment]
@bnallapeta @Roming22 do we need to disable the Automatic Merges?

**Pull request merged**